src/Groups/abelian_aut.jl

```diff
@@ -2,6 +2,7 @@ import Hecke: TorQuadMod, TorQuadModElem, TorQuadModMor
 
 export defines_automorphism
 
+@attributes TorQuadMod   # remove as soon as Hecke is patched
```

Comment:
```suggestion
@attributes TorQuadMod   # TODO: remove as soon as Hecke is patched
```
Easier to find when grepping for "TODO". Did you already fix it in Hecke?

Reply:
Yes it is fixed in Hecke.